Does not build on 64bit Haiku

```
g++ -c xHeaderListItem.cpp -iquote./  -iquote./   -O3    -o "objects.x86_64-cc8-release/xHeaderListItem.o"
xHeaderListItem.cpp:19:75: error: default argument given for parameter 3 of 'void xHeaderListItem::DrawItem(BView*, BRect, bool)' [-fpermissive]
 xHeaderListItem::DrawItem(BView *owner, BRect frame, bool complete = false)
                                                                           ^
In file included from xHeaderListItem.cpp:4:
xHeaderListItem.h:17:15: note: previous specification in 'virtual void xHeaderListItem::DrawItem(BView*, BRect, bool)' here
  virtual void DrawItem(BView *owner, BRect frame, bool complete = false);
               ^~~~~~~~
/boot/system/develop/etc/makefile-engine:297: recipe for target 'objects.x86_64-cc8-release/xHeaderListItem.o' failed
make: *** [objects.x86_64-cc8-release/xHeaderListItem.o] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -62,7 +62,7 @@ RSRCS = repliman.rsrc data.rsrc
 #	- 	if your library does not follow the standard library naming scheme,
 #		you need to specify the path to the library and it's name.
 #		(e.g. for mylib.a, specify "mylib.a" or "path/mylib.a")
-LIBS = be
+LIBS = be $(STDCPPLIBS)
 
 #	Specify additional paths to directories following the standard libXXX.so
 #	or libXXX.a naming scheme. You can specify full paths or paths relative

--- a/xHeaderListItem.cpp
+++ b/xHeaderListItem.cpp
@@ -16,7 +16,7 @@ xHeaderListItem::xHeaderListItem(const char* name, rgb_color background_colour)
 }
 
 void
-xHeaderListItem::DrawItem(BView *owner, BRect frame, bool complete = false)
+xHeaderListItem::DrawItem(BView *owner, BRect frame, bool complete)
 {
 	owner->SetLowColor(fColour);
 	owner->FillRect(frame, B_SOLID_LOW);

--- a/xListItem.cpp
+++ b/xListItem.cpp
@@ -18,7 +18,7 @@ xListItem::xListItem(const char* name, BBitmap *icon, int32 i, int32 location)
 }
 
 void
-xListItem::DrawItem(BView *owner, BRect frame, bool complete = false)
+xListItem::DrawItem(BView *owner, BRect frame, bool complete)
 {
 	rgb_color colour;
 

